<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuButtons</class>
 <widget class="QWidget" name="menuButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
<<<<<<< HEAD
     <item row="0" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_3">
=======
     <item row="1" column="0">
      <widget class="QPushButton" name="btnOrder">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>105</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>175</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>24</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border-radius: 20px;
	color : white;
	border-image: url(:/img/img/ButtonBackground.jpg);
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}</string>
       </property>
       <property name="text">
        <string>ORDER</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_5">
>>>>>>> 24f7eebdf8c0d81e98db20e676926fa1df9acbc6
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnAdmin">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>105</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>175</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>24</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border-radius: 20px;
	color : white;
	border-image: url(:/img/img/ButtonBackground.jpg);
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}</string>
       </property>
       <property name="text">
        <string>ADMIN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnBillHistory">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>105</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>175</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>24</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border-radius: 20px;
	color : white;
	border-image: url(:/img/img/ButtonBackground.jpg);
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}
</string>
       </property>
       <property name="text">
        <string>HISTORY</string>
       </property>
      </widget>
     </item>
<<<<<<< HEAD
     <item row="1" column="5">
      <spacer name="horizontalSpacer_2">
=======
     <item row="1" column="3">
      <spacer name="horizontalSpacer_3">
>>>>>>> 24f7eebdf8c0d81e98db20e676926fa1df9acbc6
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
<<<<<<< HEAD
     <item row="1" column="1">
      <spacer name="horizontalSpacer_5">
=======
     <item row="1" column="9">
      <spacer name="horizontalSpacer_2">
>>>>>>> 24f7eebdf8c0d81e98db20e676926fa1df9acbc6
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
<<<<<<< HEAD
     <item row="3" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnOrder">
=======
     <item row="1" column="10">
      <widget class="QPushButton" name="btnlogin">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>105</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>310</width>
         <height>175</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border-radius: 20px;
	color : white;
	border-image: url(:/img/img/ButtonBackground.jpg);
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}</string>
       </property>
       <property name="text">
        <string>Login Report</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QPushButton" name="btnStaff">
>>>>>>> 24f7eebdf8c0d81e98db20e676926fa1df9acbc6
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>105</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>175</height>
        </size>
       </property>
<<<<<<< HEAD
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>24</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
=======
>>>>>>> 24f7eebdf8c0d81e98db20e676926fa1df9acbc6
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border-radius: 20px;
	color : white;
	border-image: url(:/img/img/ButtonBackground.jpg);
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}</string>
       </property>
       <property name="text">
<<<<<<< HEAD
        <string>ORDER</string>
       </property>
      </widget>
     </item>
=======
        <string>Staff </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
>>>>>>> 24f7eebdf8c0d81e98db20e676926fa1df9acbc6
     <item row="1" column="6">
      <widget class="QPushButton" name="btnServer">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>105</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>175</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>24</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border-radius: 20px;
	color : white;
	border-image: url(:/img/img/ButtonBackground.jpg);
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}</string>
       </property>
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
<<<<<<< HEAD
=======
     <item row="1" column="5">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="7">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
>>>>>>> 24f7eebdf8c0d81e98db20e676926fa1df9acbc6
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
